src/fsharp/FSharp.Compiler.Service/FSharp.Compiler.Service.netcore.fsproj

```diff
@@ -0,0 +1,615 @@
+<Project Sdk="FSharp.NET.Sdk;
+Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <VersionPrefix>11.0.8</VersionPrefix>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <DefineConstants>
+      $(DefineConstants);
```

Comment:
@dsyme my bad, i asked for that because is more merge friendly. 
sry @baronfel :D